Стр/15.1/15.2.py

```diff
@@ -1,2 +1,7 @@
 """ Напиши программу, которая берет два целых числа
 в качестве ввода и выводит их сумму. """
+
+A = int(input())
+B = int(input())
+C = A + B
+print(C)
```

Comment:
```suggestion
A = int(input())
B = int(input())
print(A + B)
```Код можно сделать короче